ddp-workspace/projects/ddp-dsm-ui/src/app/navigation/navigation.component.html

```diff
@@ -15,7 +15,8 @@
         </ul>
       </li>
       <li class="dropdown" *ngIf="getAuth().authenticated()
-        && (hasRole().allowToViewSampleLists() || hasRole().allowedToHandleSamples() || hasRole().allowedToViewReceivingPage() || hasRole().allowedToDiscardSamples() || hasRole().allowedToExitParticipant())">
+        && (hasRole().allowToViewSampleLists() || hasRole().allowedToHandleSamples() || hasRole().allowedToViewReceivingPage() || hasRole().allowedToDiscardSamples() || hasRole().allowedToExitParticipant()
+        || hasRole().allowedToDoOrderSequencing() || hasRole().viewSeqOrderStatus)">
         <a class="dropdown-toggle" data-toggle="dropdown" href="#"><b>Samples</b>
```

Comment:
How does 'allowedToDoOrderSequencing()'  fit into this feature?

Reply:
Is viewSeqOrderStatus a property or a function?

Reply:
I created a test user with only this new permission and participant list persmission, and realized that they won't see the `Samples` menu since it only checked for samples and kit shipping access, while people with either of `allowedToDoOrderSequencing` or `viewSeqOrderStatus` should be able to see the `Clinical Orders` menu point in this drop down. That's why I corrected this.

Reply:
@otchet-broad `viewSeqOrderStatus` is a get function

Reply:
I assumed the issue here was the lack of parens after 'viewSeqOrderStatus'?

Reply:
Thank you for catching this! It was a naming issue where both the variable and the function had the same name. I fixed it now